public/app/features/plugins/admin/components/PluginDetailsDeprecatedWarning.tsx

```diff
@@ -16,13 +17,19 @@ export function PluginDetailsDeprecatedWarning(props: Props): React.ReactElement
   let deprecationMessage = `This ${plugin.type} plugin is deprecated and has been removed from the catalog. No further updates will be made to the
   plugin.`;
 
-  if (plugin.details?.statusContext) {
-    deprecationMessage += ` More information: ${plugin.details.statusContext}`;
-  }
-
   return isWarningVisible ? (
     <Alert severity="warning" title="Deprecated" className={className} onRemove={() => setDismissed(true)}>
       <p>{deprecationMessage}</p>
+
+      {/* Additional contextual deprecation message supporting markdown */}
+      {plugin.details?.statusContext && (
+        <p
```

Comment:
The markdown will likely render `p` tags for the text and this will generate invalid HTML. Can you use a div here instead?

Reply:
Good catch! 👍